quick_start/requirements.txt

```diff
@@ -0,0 +1,4 @@
+azure-kusto-data==3.0.1
+azure-kusto-ingest==3.0.1
+inflection==0.5.1
+tqdm==4.64.0
```

Comment:
For our packages, you can set it to non breaking versions:
For others, you can set it to minor versions up:
```
azure-kusto-data>=3.0.1,<4.0.0
azure-kusto-ingest>=3.0.1,<4.0.0

inflection~=0.5.1
tqdm~=4.64.0
```